src/main/java/ch/njol/skript/SkriptEventHandler.java

```diff
@@ -255,9 +259,20 @@ static void registerBukkitEvents() {
 			if (e.equals(PlayerInteractAtEntityEvent.class) || e.equals(PlayerArmorStandManipulateEvent.class)) {
 				continue; // Ignore, registered above
 			}
-			if (!registeredEvents.contains(e)) { // Check if event is registered
+
+			HandlerList handlerList;
+
+			try {
+				Method getHandlerListMethod = e.getMethod("getHandlerList");
+				handlerList = (HandlerList) getHandlerListMethod.invoke(e);
+			} catch (Exception exception) {
+				handlerList = null;
```

Comment:
I think it'd be a good idea to print the exception here, but only in a high verbosity, for example `Skript.logVeryHigh()`.